UIView+MGBadgeView.m

```diff
@@ -95,6 +117,43 @@ - (void)setBadgeValue:(NSInteger)badgeValue {
     }
 }
 
+- (void)setBadgeText:(NSString *)badgeText {
+    if(![(badgeText ?: @"") isEqualToString:(_badgeText ?: @"")]) {
```

Comment:
would be nicer if you can bring this in a separated method, maybe a static function like MGEqualObjects(obj1, obj2) which checks nullability. Or otherwise use the normal isEqualToString and before that, just check if they are both nil, and in that case return.

Reply:
Ok, I've simplified it to simply
```
    if(_badgeText != badgeText) {
        _badgeText = badgeText;
```

And tested, it has the same result.

Reply:
No dude sorry, even if they are NSString, I don't like to compare objects in this way. I gave you a suggest on how to approach, I don't like shortcut :)

```
static BOOL MGEqualObjects(NSString *string1, NSString *string2) {
    return (!string1 && !string2) || [string1 isEqualToString:string2];
}
```